site/src/pages/AIGovernancePage/RequestLogsPage/filter/filter.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+	type UseFilterMenuOptions,
+	useFilterMenu,
+} from "components/Filter/menu";
+import {
+	SelectFilter,
+	type SelectFilterOption,
+} from "components/Filter/SelectFilter";
+
+const AIBRIDGE_PROVIDERS: SelectFilterOption[] = [
+	{
+		label: "OpenAI",
+		value: "openai",
+	},
+	{
+		label: "Anthropic",
+		value: "anthropic",
+	},
+];
+
+export const useProviderFilterMenu = ({
+	value,
+	onChange,
+	enabled,
+}: Pick<UseFilterMenuOptions, "value" | "onChange" | "enabled">) => {
+	return useFilterMenu({
+		id: "provider",
+		getSelectedOption: async () =>
+			AIBRIDGE_PROVIDERS.find((option) => option.value === value) ?? null,
```

Comment:
```suggestion
			AIBRIDGE_PROVIDERS.find((option) => option.value === value),
```

why are you coercing it to `null` just to immediately coerce it _back_ to `undefined`?